### PR TITLE
Update docs for 1971

### PR DESCRIPTION
**Pull request by kfu02**
## Description
@sid-parikh and I forgot to update the docs to reflect the new operation of make run-sim introduced by #1971 . Fixed here.

### CODE REVIEW

**Review by sid-parikh — APPROVED** [review]
Looks good. Thanks for remembering

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -143,10 +144,12 @@ terminal, you can launch sim with:
...
 .. code-block:: bash
 
-   . ./source.bash make run-sim
+   . ./source.bash 
+   make run-sim
 
 ``source.bash`` is an alias for the two source commands you saw above, and
-``make run-sim`` is an alias for ``ros2 launch rj_robocup sim.launch.py``.
+``make run-sim`` will launch both ER-Force's Framework (the physics simulator)
+and our stack (``ros2 launch rj_robocup sim.launch.py``).
```
Comment (sid-parikh) [inline docs/source/getting_started.rst]: Maybe add a line mentioning the need to Ctrl-C twice? Might not be obvious to beginners

Comment (kfu02) [reply docs/source/getting_started.rst]: will do, thanks!

**Comment by kfu02** [comment]
replaced by #1981

**Pull request closed without merging**